oca_dependencies.txt

```diff
@@ -13,3 +13,4 @@
 #
 # To provide both the URL and a branch, use:
 # sale-workflow https://github.com/OCA/sale-workflow branchname
+connector https://github.com/OCA/connector 11.0
```

Comment:
why?

Reply:
component and component_event modules comes from connector repo

Reply:
ah yes, that makes a nice circular dependency loop between connector and queue. Not sure if it is an issue in practice.

Maybe it's lighter to overwrite create/write manually for this simple use case? What do you think @lmignon @guewen ?

Reply:
Indeed, It could be lighter to only overwrite 'create/write' methods into the mail.mail to avoid this dependency. On the other hand the code looks clean and is easy to read when using the component.event addon to add a hook on the create/write method. The more I use components, the more I'm convinced that we should remove the dependency on connector to make it more generic. But it's an other debate.  Since the dependency is only required for this unique use into the addon I'm in favour to remove-it.